hikari/impl/stateful_cache.py

```diff
@@ -1598,3 +1605,32 @@ def update_voice_state(
         cached_voice_state = self.get_voice_state(voice_state.guild_id, voice_state.user_id)
         self.set_voice_state(voice_state)
         return cached_voice_state, self.get_voice_state(voice_state.guild_id, voice_state.user_id)
+
+    def clear_messages(self) -> cache.CacheView[snowflakes.Snowflake, messages.PartialMessage]:
```

Comment:
Random question but what's the point of this cache store, what's the use case for storing diff state updates for messages (which is what a PartialMessage is) alongside full messages rather than just full messages? (also wouldn't this potentially be replacing full message objects with message diff update objects?)